cola/widgets/status.py

```diff
@@ -524,6 +526,9 @@ def _set_subtree(self, items, idx,
             parent.addChild(treeitem)
         self._expand_items(idx, items)
         self.blockSignals(False)
+        if prefs.status_show_totals(self.context):
+            parent_title = parent.text(0).split()[0]
+            parent.setText(0, '%s (%s)' % (parent_title, len(items)))
```

Comment:
What happens if this block of code runs more than once?  It avoids problems by calling `split()` to always take the first word only, but that seems a little too complicated.

All of the `_set_subtree` callers "know" what the correct title should be, so it'd be better to just pass down an explicit title instead of getting it from the `parent.text()`.  We can still do the `'%s (%s)'` thing here, but it'll at least separate the logic from being tied to details of what the message looks like.

For example, maybe the internationalized text will have more than one word, so it's better to make it generic.